src/main/java/org/gridsuite/network/map/dto/mapper/GeneratorInfosMapper.java

```diff
@@ -42,8 +43,10 @@ private GeneratorInfosMapper() {
     }
 
     public static ElementInfos toData(Identifiable<?> identifiable, InfoTypeParameters infoTypeParameters) {
+        boolean loadRegulatingTerminals = Optional.ofNullable(infoTypeParameters.getOptionalParameters().get(QUERY_PARAM_LOAD_REGULATING_TERMINALS))
+            .map(Boolean::valueOf).orElse(false);
```

Comment:
to be compatible with actual result
```suggestion
            .map(Boolean::valueOf).orElse(true);
```

Reply:
same